src/app/api.ts

```diff
@@ -1075,6 +1108,98 @@ export function newAPI(): API {
     return secrets;
   }
 
+  async function createSecretProjects(
+    username: string,
```

Comment:
why username? The api wants a `projectref`/`projectgroupref`. See the backend code.